djangoevents/utils.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+from .domain import BaseEntity
+from .domain import BaseAggregate
+from .domain import DomainEvent
+
+
+def list_concrete_aggregates():
```

Comment:
Yeah, that's very good point. We might need something like auto-imports we did for handlers (https://github.com/ApplauseOSS/djangoevents/blob/devel/djangoevents/apps.py)